src/components/builder/prefill/prefill.stories.tsx

```diff
@@ -130,3 +130,37 @@ ChangingPluginClearsAttribute.play = async ({canvasElement}) => {
     await expect(canvas.queryByText('Plugin 1, attribute 2')).toBeInTheDocument();
   });
 };
+
+export const ChangingIdentifierRole = Template.bind({});
+ChangingIdentifierRole.parameters = {
+  formik: {
+    initialValues: {
+      prefill: {
+        plugin: 'plugin-2',
+        attribute: 'plugin-2-attribute-1',
+        identifierRole: 'main',
+      },
+    },
+  },
+};
+ChangingIdentifierRole.play = async ({canvasElement}) => {
+  const canvas = within(canvasElement);
+
+  await waitFor(async () => {
```

Comment:
I don't think we need the `waitFor` here because the options are not retrieved async but sync, while the prefill plugins and attributes are retrieved from an API endpoint.

Reply:
Mhmm if I remove it, the interaction tests fail :thinking: